### DISCUSSION TITLE
apt-get not working?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5325] [General] [answered]
Hi, i am trying to experiment with how the order of the NICs (ens??). It looks like to see the order i have to edit /etc/network/interfaces. 
 I am just trying to see if I can change the ens?? Order. It looks like I just need to edit "/etc/network/interfaces", but i don’t know how to open it. I tried $sudo nano and $sudo cat. I also tried $sudo apt-get nano, but apt-get is not found?
Current version - securityonion-2.3.61-STENODOCKER.iso
CentOS 7
EVAL
Show so-status  
![image](https://user-images.githubusercontent.com/5569263/131199146-6babe0be-af1f-4af2-9a04-ea5a9aa366b8.png)
![image](https://user-images.githubusercontent.com/5569263/131199232-340f883f-4986-4708-bb2e-f618453d847d.png)

Salt-call state.highstate – have to study up on this.

![image](https://user-images.githubusercontent.com/5569263/131199676-bb86eafc-0fc2-4f33-90bf-ac3e1873a2ff.png)

Soup.log – see PDF (is there a link that explains how to transfer a log file from so to my host)
grid status - from what I see, you have to get on SOC for this? I am not able to get on the SOC yet, I get errors. But please to refer to another post I put out with this one?? #5063
security sudo stuff.pdf
here is my install_summary
![image](https://user-images.githubusercontent.com/5569263/131199757-a9cf6bf8-afd4-4a90-a15e-99a23c8537f5.png)
 
Soup.log –  I am trying to connect to github, this is one reason why I am looking at the network interfaces
![image](https://user-images.githubusercontent.com/5569263/131200025-199e7984-2bde-4332-b8d7-531f6879b3c4.png)

grid status - from what I see, you have to get on SOC for this? I am not able to get on the SOC yet, I get errors. But please to refer to another post I put out with this one?? https://github.com/Security-Onion-Solutions/securityonion/discussions/5063  

thanks for any advice

**Comment by greatapoc** [comment, marked as answer]
CentOS doesn't use apt-get it uses yum.

**Reply by iqworks** [reply]
thanks gretapoc, it makes sense that you should look at the operating system and find their commands.

**Comment by iqworks** [comment]
Ahh, thanks
